app/src/main/res/layout/activity_login.xml

```diff
@@ -7,99 +7,113 @@
     tools:context=".LoginActivity">
 
```

Comment:
In questo layout, non era meglio lasciare i constraint come avevo fatto io, così se lo schermo veniva girato era già fatto, cosi invece bisogna fare un nuovo layout

Reply:
cosa intendi?